### PR TITLE
auto update npm deps and devDeps from tpl

### PR DESCRIPTION
**Pull request by j-d-carmichael**

### CODE REVIEW

**Comment by j-d-carmichael** [comment]
https://github.com/acr-lfr/generate-it/issues/113

File: src/commander.ts [hunk 1]
```diff
@@ -39,6 +40,7 @@ export default (inputArgsArray: string[]): CommanderResponse => {
     .requiredOption('-t, --template <helpers>', 'Full URL to a public git repo, eg github')
     .option('--dont-update-tpl-cache', 'If the given git url is already cached does not attempt to update', false)
     .option('--dont-run-comparison-tool', 'Skips the stub file comparison tool and version cleanup', false)
+    .option('-u --update-dependencies-from-tpl', 'Run the npm install scripts inline with the tpl package.json opposed to displaying for manual update', false)
```
Comment (p-mcgowan) [inline src/commander.ts:43]: missing comma?

**Review by p-mcgowan — COMMENTED** [review]
looks cool - will want to do a test pull in the morning to make sure, but seems like it should be good.

**Comment by j-d-carmichael** [comment]
True good spot.. weirdly it still works though. I'll commit the , either way.

**Comment by j-d-carmichael** [comment]
how's she looking @p-mcgowan ? Good to go?

**Comment by p-mcgowan** [comment]
bewty - looks good. stdout is ugly but seems like it works with both short and long opt.

some time later maybe we could add support in the nodegen file, but for first pass this is great

**Comment by j-d-carmichael** [comment]
Beauty is in the eye of the beholder 😅

Yea via spawn would be better too, I'll probs send another pr in for that later with your idea for the RC fine too 👍

**Pull request merged**